src/forms/Select.jsx

```diff
@@ -56,15 +57,18 @@ export class SelectInput extends React.PureComponent<Props> {
 			: {};
 
 		return (
-			<SwarmSelect
-				id={id || name}
-				label={label}
-				name={name}
-				error={error}
-				helperText={helperText}
-				{...requiredProps}
-				{...other}
-			/>
+			<React.Fragment>
+				{label && <FieldLabel>{label}</FieldLabel>}
+				{helperText && <FieldHelper>{label}</FieldHelper>}
+				<SwarmSelect
+					id={id || name}
+					label={label}
```

Comment:
We probably shouldn't be passing the label here too. It's a bit misleading / confusing. I can't remember if the `swarm-ui` select element expects a label prop, but if it does, we should remove that and clean this up.